src/main/webapp/script.js

```diff
@@ -54,3 +57,8 @@ function resetForm(id) {
     const placeholder = id === channelId ? channelPlaceholder : businessPlaceholder;
     $("#formInput").attr("placeholder", placeholder);
 }
+
+function triggerLoading() {
```

Comment:
this is non-obvious to me what this is doing.  can you add some javadocs? (javascript docs)?